### PR TITLE
RHTAPBUGS-1151: ArgoCD Application-Controller Memory Limit

### PR DESCRIPTION
**Pull request by otaviof**
Adding more memory to the ArgoCD controller.

/cc @flacatus

### CODE REVIEW

**Comment by flacatus** [comment]
@otaviof @Roming22 will not be better to increase all the container memory like we do in Konflux:

```yaml
containers:
    - resources:
        limits:
          cpu: '2'
          memory: 6Gi
        requests:
          cpu: '1'
          memory: 3Gi
```

**Comment by Roming22** [comment]
@otaviof Flavius' comment makes sense. It might be worth it to bump the requirements until we're in a position to let the users fine tune their configuration.

**Comment by otaviof** [comment]
> @otaviof @Roming22 will not be better to increase all the container memory like we do in Konflux? Example
> 
> ```yaml
> containers:
>     - resources:
>         limits:
>           cpu: '2'
>           memory: 6Gi
>         requests:
>           cpu: '1'
>           memory: 3Gi
> ```

Sure! I'm exposing the ArgoCD resource limits on `values.yaml` so we can adapt to each situation, by default comes with the limits you suggested.

**Comment by flacatus** [comment]
LGTM

File: values.yaml [hunk 1]
```diff
@@ -15,6 +15,18 @@ openshift-gitops:
 openshift-pipelines:
   enabled: true
   channel: pipelines-1.14
+
+argoCD:
```
[the remark this answers is not in the document]
Comment (otaviof) [reply values.yaml]: Thanks, just moved into that section, please consider.

**Review by Roming22 — APPROVED** [review]
/lgtm

**Pull request merged**